client/src/app/components/header/header.component.ts

```diff
@@ -1,15 +1,30 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
+import { UserSearch } from 'src/app/interfaces/userSearch';
+
 import { StoreFacadeService } from '../../store/store-facade.service';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
 })
-export class HeaderComponent {
+export class HeaderComponent implements OnInit {
   @Input() userPhotoURL: string;
+  userSearch$ = this.storeFacade.userSearch$;
+  showSearchBar = false;
   constructor(private storeFacade: StoreFacadeService) {}
+  ngOnInit(): void {
+    this.storeFacade.loadUserSearchList();
 
+    this.userSearch$.subscribe((userList: UserSearch[]) => {
+      // Loading search user list
+      if (!!userList) {
+        this.showSearchBar = true;
+      } else {
+        this.showSearchBar = false;
+      }
+    });
+  }
```

Comment:
This can be changed to:
this.showSearchBar = !!userList ? true : false;

Reply:
it could actually just be ```this.showSearchBar = !!userList```

Reply:
> it could actually just be `this.showSearchBar = !!userList`

that's true 😄